.github/workflows/automationTemplate.yaml

```diff
@@ -85,46 +75,85 @@ jobs:
       
       - name: Install jq
         run: |
+          if [ ${{ env.CANCEL_WORKFLOW }} = true ]; then
+            echo "No new ${{ inputs.platformName }} SDK versions detected. Canceling workflow..."
```

Comment:
Nit: please consider moving this echo to the previous step where we actually detected that no SDK versions existed.
Else people might not expect to find this information in the "Install JQ" step. Also, if for any reasons in the future we add new reasons for the flow to be canceled, then this message would become misleading.

Reply:
Makes sense! I've updated too.

Reply:
Thanks!